Assets/MRTK/SceneSystemResources.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Woah I guess I somehow missed that this was a top-level MRTK folder. That feels SUPER weird to me. How do our .unitypackages work with that?

Reply:
Can it live in Services/SceneSystem? Or even wherever it used to live, but we just move them out before packaging for UPM so they aren't duplicated? Or am I missed a reason they needed to be standalone?

Reply:
not easily with the packaging process.... i will confirm that our unitypackage files do the right thing

Reply:
fixed by #8780